### PR TITLE
Remove HHVM

### PR DESCRIPTION
**Pull request by Nyholm**
It causes weird errors like: https://travis-ci.org/api-php/localise.biz/jobs/182756476

### CODE REVIEW

**Comment by sagikazarmark** [comment]
Strange. Allow failures until we find out what's wrong?

**Comment by Nyholm** [comment]
Right now I think it is a new composer thing.. We specify that the code needs PHP7 or above. HHVM is not PHP 7 or above. 

I think HHVM is equal to PHP5.6 is composers eyes.

**Comment by sagikazarmark** [comment]
Yes, but if the 7.0 code actually works on HHVM, I would just put it to allow_failure, so that after composer works with HHVM as 7.0 we can just enable it. WDYT?

File: .travis.yml [hunk 1]
```diff
@@ -24,6 +24,8 @@ matrix:
     include:
         - php: 7.0
           env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" COVERAGE=true TEST_COMMAND="composer test-ci"
+    allow_failures:
+        - php: hhvm
```
Comment (sagikazarmark) [inline .travis.yml:28]: Maybe add fast_forward as well, so that Travis doesn't wait for the allowed to fail builds to announce the result.

Comment (Nyholm) [reply .travis.yml:28]: `fast_finish:true` is already enabled.

**Pull request merged**